docs/ide.md

```diff
@@ -37,3 +37,17 @@ We recommend additionally setting the following in your `.vscode/settings.json`
     }
 }
 ```
+
+## Vim
+
+### LLVM/MLIR settings
+
+Check out the official instructions for [LLVM](https://github.com/llvm/llvm-project/blob/main/llvm/utils/vim/README)
+and [MLIR](https://github.com/llvm/llvm-project/blob/main/mlir/utils/vim/README)
+settings to enable syntax highlighting and other goodies.
+
+### IDE-like features
+
+Check out the official [documentation for
+`clangd`](https://releases.llvm.org/9.0.1/tools/clang/tools/extra/docs/clangd/Installation.html)
+to enable features like autocompletion, go to definition, etc.
```

Comment:
We might want to highlight the clangd public LLVM index -- it;'s a good QoL improvement

Reply:
Interesting! I haven't found myself needing that and don't know anything about it, feel free to send a change to add that.